site/app/templates/admin/admin_gradeable/AdminGradeableDates.twig

```diff
@@ -59,6 +59,12 @@
         <div id="release_date_container" class="release_date_date" {{ gradeable.hasReleaseDate() ? '' : 'hidden' }}>
             <input name="grade_released_date" id="date_released" class="date_picker" type="text" value="{{ gradeable.getGradeReleasedDate()|date(date_format, timezone_string) }}">
             <label for="date_released">Grades Released Date (grades will be visible to students)</label>
+        
+            <br />
+
+            <p style="max-width:600px;" id="checkpoint-numeric-gradeables-message">
```

Comment:
Why does this need a max width?

Reply:
I added that so that that it would match the style of the `#late-days-message` note (since the note is relatively long, this makes it wrap to the next line and not span horizontally).